pulpcore/pulpcore/app/models/content.py

```diff
@@ -138,6 +138,27 @@ def natural_key(self):
         return tuple(getattr(self, f) for f in self.natural_key_fields())
 
 
+class SingleArtifactContent:
```

Comment:
Why is this a _mixin_ instead of a subclass of `Content`?  Generally, I'm opposed to _mixins_.

Why not add these convenience properties to `Content`?

Reply:
Will make SingleArtifactContent a subclass 👍 

The reason I didn't want to add these to `Content` is that I didn't want plugin writers to accidentally call them if they're dealing with Content that may have multiple artifacts.

Reply:
I'm cool with adding them to `Content` though and maybe being explicit in the docstring that they're not for use with Content that can have multiple artifacts?